assignments/prime_challenge.md

```diff
@@ -0,0 +1,24 @@
+## Prime Challenge
+
+Given a number between 1-100, return true if the number is a prime.
+
+## Resources
+
+To prompt the user for a number to check, you can do something like this:
+
+print "What number would you like to check?  "
+number = gets.chomp.to_i
+
+You can check your work here http://whatis.techtarget.com/definition/prime-number
+
+## Hint
+
+https://stackoverflow.com/questions/1801391/what-is-the-best-algorithm-for-checking-if-a-number-is-prime
```

Comment:
I think this might actually just scare the hell out of them. :joy: What did you want them to get out of this post?

Reply:
Honestly, it's just the answer in a different language. This is the one I was thinking of: https://stackoverflow.com/a/1801446/8856470. Open to figuring out some kind of hints that don't tell the answer. :thinking_face: